### PR TITLE
add cleanup cmd

### PR DESCRIPTION
**Pull request by wei-lee**
to verify:

* checkout the PR and build the cli
* Run:
   ```
    ./delorean report cleanup --config-file=./configurations/cleanup-reports.yaml
   ```
* You should see some outputs and some objects will be moved to the `archive/` sub dir in the bucket.
* Remove the objects back to the root dir manually.

### CODE REVIEW

**Comment by b1zzu** [comment]
:eyes:

**Comment by wei-lee** [comment]
@b1zzu are you able to review this PR yet?

File: cmd/cleanupReports.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+		}
+		if ok {
+			fmt.Println(fmt.Sprintf("[%s] Object %s has matched tags and will be moved", bucket, *o.Key))
+			toCopy = append(toCopy, o)
+		} else {
+			fmt.Println(fmt.Sprintf("[%s] Skip object %s as it doesn't have the required tags", bucket, *o.Key))
+		}
+	}
+	copied := c.copyObjects(ctx, bucket, archiveFolderName, toCopy)
+	if err = c.deleteObjects(ctx, bucket, copied); err != nil {
```
Comment (b1zzu) [inline cmd/cleanupReports.go:136]: maybe we should delete the object only if the copy succeeded otherwise we risk to permanently remove it

Comment (wei-lee) [reply cmd/cleanupReports.go:136]: `copied` field will only contain objects that are copied successfully.

**Pull request merged**